qiita_db/test/test_processing_job.py

```diff
@@ -948,5 +953,27 @@ def test_remove_error(self):
             tester.remove(tester.graph.edges()[0][0])
 
 
+@qiita_test_checker()
+class ProcessingJobDuplicated(TestCase):
```

Comment:
Is there a specific reason why you've created a different class here? That triggers a new reset of the DB, it will be great if we can avoid a reset of DB unless strictly necessary.

Reply:
As we are setting all the jobs to error the other tests are failing. I think we have 2 options: A. reset back but it seemed ugly to me or B. have it as a different class so the reset happens, this seemed better to me

Reply:
Nah, it's okay - I just wanted to make sure that there was an actual reason to create the class.